### DISCUSSION TITLE
公式前导符如何添加？

### DISCUSSION THREAD

**Discussion by btob2000** [sjtug/SJTUThesis#920] [提问]
请问公式前导符如何添加？
类似图片这种：
![image](https://user-images.githubusercontent.com/85401989/231989378-c8567d3c-73d6-495a-9476-3414128bea35.png)

**Comment by LogCreative** [comment]
可以考虑使用 `dotseqn` 宏包，但是似乎只对 `equation` 环境有效。[^1][^2]
```latex
\usepackage{dotseqn}
```

[^1]: https://tex.stackexchange.com/questions/603473/how-to-fill-dots-between-equation-and-tagequation-numbers-in-align-environment
[^2]: https://mirrors.sjtug.sjtu.edu.cn/CTAN/macros/latex/contrib/dotseqn/dotseqn.pdf
